book_source/source/2darrays.md

```diff
@@ -0,0 +1,237 @@
+# <i class="fas fa-book fa-fw"></i> 2DArrays
+
+## 2DArray Fundamentals
+
+An 2D array is very similar to an 1D array, or just an array as we have learned, only that instead of storing primiative types, it stores another array. In other words, a 2D array is an array where each element contains another array. Commonly this is represented/used when dealing with images, graphs, tables, and various other 2D representations.
+
```

Comment:
I think introducing the idea of "rows" vs. "columns" would make more sense to come here, before we get into syntax!